config/default/core.entity_view_display.block_content.uiowa_card.default.yml

```diff
@@ -76,4 +68,5 @@ hidden:
   entity_print_view_epub: true
   entity_print_view_pdf: true
   entity_print_view_word_docx: true
+  field_uiowa_card_button_display: true
```

Comment:
was this intentional? just checking

Reply:
Yeah. I think that is field that just determines the whether the button displays and we don't want to actually show it.